### PR TITLE
Add sentinelhub, bolt-python, thunder-[python, registration], eolearn-[core, visualization, ml-tools, features, io], s2cloudless recipes

### PR DESCRIPTION
**Pull request by mwilson8**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by mwilson8** [comment]
@dcunn @oblute @rluria14 @benhuff will you be maintainers with me?

**Comment by mwilson8** [comment]
@conda-forge-admin please rerender

**Comment by mwilson8** [comment]
@conda-forge-admin please rerender

**Comment by mwilson8** [comment]
@conda-forge/staged-recipes  @conda-forge/help-python please review

**Comment by beckermr** [comment]
Wow this one is impressive! It will take us a bit more time than usual to comment here I expect. Thank you for the contribution!

File: recipes/eo-learn-ml-tools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - eo-learn-core
+    - matplotlib
```
[the remark this answers is not in the document]
Comment (beckermr) [reply recipes/eo-learn-ml-tools/meta.yaml]: See discussion here: https://github.com/conda-forge/conda-forge.github.io/issues/933

If the code does not pass with this change, feel free to ignore. It is correct, but X11 can be missing in the CI which causes an issue in some cases.

File: recipes/eo-learn-visualization/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - eo-learn-core
+    - python-graphviz
+    - jinja2
+    - matplotlib
```
[the remark this answers is not in the document]
Comment (beckermr) [reply recipes/eo-learn-visualization/meta.yaml]: Ditto here

**Comment by mwilson8** [comment]
@chrisburr @beckermr changes applied

**Pull request merged**